lms/static/lms/js/spec/main.js

```diff
@@ -679,6 +679,7 @@
     });
 
     testFiles = [
+        'course_experience/js/spec/course_outline_factory_spec.js',
```

Comment:
Oh yeah, this could totally be dynamic. I don't know about configuring Karma specifically but most JS test frameworks nowdays let you set a regex for finding tests (e.g., `.*_spec\.js`). This probably just exists as a holdover from JSTestTool.